src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Filter.cs

```diff
@@ -145,7 +145,7 @@ public override bool IsServerDelegatable(CallNode callNode, TexlBinding binding)
             Contracts.AssertValue(callNode);
             Contracts.AssertValue(binding);
 
-            if (!CheckArgsCount(callNode, binding))
+            if (!CheckArgsCount(callNode, binding, DocumentErrorSeverity.Moderate))
```

Comment:
>DocumentErrorSeverity.Moderate [](http://example.com/codeflow?start=51&length=30)

Why are we changing this?

Reply:
A warning should not prevent delegation.